akka-docs/rst/java/distributed-pub-sub.rst

```diff
@@ -67,42 +84,65 @@ If all the subscribed actors have the same group name, then this works just like
 If all the subscribed actors have different group names, then this works like
 normal ``Publish`` and all messages are broadcast to all subscribers.
 
-You register actors to the local mediator with ``DistributedPubSubMediator.Put`` or
-``DistributedPubSubMediator.Subscribe``. ``Put`` is used together with ``Send`` and
-``SendToAll`` message delivery modes. The ``ActorRef`` in ``Put`` must belong to the same
-local actor system as the mediator. ``Subscribe`` is used together with ``Publish``.
-Actors are automatically removed from the registry when they are terminated, or you
-can explicitly remove entries with ``DistributedPubSubMediator.Remove`` or
-``DistributedPubSubMediator.Unsubscribe``.
+.. _distributed-pub-sub-send-java:
 
-Successful ``Subscribe`` and ``Unsubscribe`` is acknowledged with
-``DistributedPubSubMediator.SubscribeAck`` and ``DistributedPubSubMediator.UnsubscribeAck``
-replies.
+Send
+----
 
-Cluster members with status :ref:`WeaklyUp <weakly_up_java>`, if that feature is enabled,
-will participate in Distributed Publish Subscribe, i.e. subscribers on nodes with 
-``WeaklyUp`` status will receive published messages if the publisher and subscriber are on
-same side of a network partition.
+This is a point-to-point mode where each message is delivered to one destination,
+but you still does not have to know where the destination is located.
+A typical usage of this mode is private chat to one other user in an instant messaging
+application. It can also be used for distributing tasks to registered workers, like a 
+cluster aware router where the routees dynamically can register themselves.
+
+The message will be delivered to one recipient with a matching path, if any such
+exists in the registry. If several entries match the path the message will be sent
+via the supplied ``RoutingLogic`` (default random) to one destination. The sender() of the
+message can specify that local affinity is preferred, i.e. the message is sent to an actor
+in the same local actor system as the used mediator actor, if any such exists, otherwise
+route to any other matching entry. 
 
-A Small Example
----------------
+You register actors to the local mediator with ``DistributedPubSubMediator.Put``.
+The ``ActorRef`` in ``Put`` must belong to the same local actor system as the mediator.
+The path without address information is the key to which you send messages.
+On each node there can only be one actor for a given path, since the path is unique
+within one local actor system.
 
-A subscriber actor:
+You send messages by sending ``DistributedPubSubMediator.Send`` message to the
```

Comment:
Should it mention what happens if the same path has been registered on multiple nodes?